PandasWorkshop.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>Awesome! Thank you! This is a great walkthrough and you did a great job choosing what topics to cover. I have a couple comments above on how to make things a bit more accessible and idiomatic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/orioncohen/metal-bands-by-nation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>